shared-data/default-theme/html/jsapi/global/global.js

```diff
@@ -5,38 +5,12 @@ Mailpile.fix_url = function(url) {
   return url;
 }
 
-Mailpile.go = function(url) {
```

Comment:
This method has been [monkey-patched](https://github.com/grekko/Mailpile/blob/4c0384fbfed1d44bd7761818176822c93e1fd79e/shared-data/contrib/autoajax/autoajax.js#L234) by autoajax and therefore never been executed again.